test/integration/lib/test-actions/index.ts

```diff
@@ -0,0 +1,136 @@
+import assert from 'assert'
+import { MockASE } from '../mock-ase'
+import { parseCookies } from '../utils'
+import { WalletAddress, PendingGrant } from '@interledger/open-payments'
+import { AdminActions, createAdminActions } from './admin'
+import { OpenPaymentsActions, createOpenPaymentsActions } from './open-payments'
+
+export interface TestActionsDeps {
+  sendingASE: MockASE
+  receivingASE: MockASE
+}
+
+export interface TestActions {
+  consentInteraction(
+    outgoingPaymentGrant: PendingGrant,
+    senderWalletAddress: WalletAddress
+  ): Promise<void>
+  consentInteractionWithInteractRef(
+    outgoingPaymentGrant: PendingGrant,
+    senderWalletAddress: WalletAddress
+  ): Promise<string>
+  admin: AdminActions
+  openPayments: OpenPaymentsActions
+}
+
+export function createTestActions(deps: TestActionsDeps): TestActions {
+  return {
+    consentInteraction: (outgoingPaymentGrant, senderWalletAddress) =>
+      consentInteraction(deps, outgoingPaymentGrant, senderWalletAddress),
+    consentInteractionWithInteractRef: (
+      outgoingPaymentGrant,
+      senderWalletAddress
+    ) =>
+      consentInteractionWithInteractRef(
+        deps,
+        outgoingPaymentGrant,
+        senderWalletAddress
+      ),
```

Comment:
maybe we can pull these out into identity provider actions (or something along those lines)

Reply:
We could. Although I was thinking it would be good to let these just be miscellaneous ones so we dont have to categorize everything. The admin api and open payments api are obvious but beyond that IDK how helpful it is to add the extra scope.

On a similar note, I originally was thinking I could do `AdminTestActions`, `OpenPaymentTestActions`, etc. instead of everything scoped to 1 `TestActions`. But then wondered where to throw these consent interaction ones. I guess `IdentityProviderTestActions` could be the answer but I didn't necessarily want to think of a scope for every potential new one.